task31/task31.cpp

```diff
@@ -0,0 +1,63 @@
+// task31.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+#include <vector>
+#include <map>
+#include <algorithm>
+
+/*
+��� ������ ������� ����, � ������� � ������ ����� ������� ��������
+����� VS ��������� �����, ����� ����� ������� �����, � ������� ������ ����������.
+����� ����� ������� ��� ������� (PR) � ������ ���.
+*/
+
+
+int findNonRepeatableNumber(std::vector<int>& nums) {
+    bool repeatableFlag = false;
+    int foundNonRepeatableNumber = -999;
+    for (int i = 0; i < nums.size(); i++) {
+        repeatableFlag = true;
+        for (int j = 0; j < nums.size(); j++) {
+            if (i == j) {
+                continue;
+            }
+            if (nums[i] == nums[j]) {
+                repeatableFlag = false;
+                break;
+            }
+        }
+
+        if (repeatableFlag == 1) {
+            foundNonRepeatableNumber = nums[i];
+            break;
+        }
+    }
+    if (repeatableFlag == false) {
+        throw std::runtime_error("There is no non repeatable number");
```

Comment:
it is not the best way to throw exception in such case. because it is normal situation

Reply:
i agree. done. thank you